pywwt/annotation.py

```diff
@@ -13,30 +13,34 @@
 
 class Annotation(HasTraits):
     """
-    Base class for annotations which provides some settings common to all shapes.
+    Base class for annotations which provides settings common to all shapes.
     """
 
     shape = None
+    """
+    To be specified in the individual shape classes.
+    """
 
     label = Unicode('', help='Contains descriptive text '
-                             'for the annotation').tag(wwt='label')
+                    'for the annotation (:class:`str`)').tag(wwt='label')
```

Comment:
The previous form of indentation was actually correct since it shows better that the string is continued.

Reply:
I don't think you need the ``:class:`` for built-in types, just the backticks are enough.

Reply:
On your first comment, do you mean that I should indent strings on the following line to the same level as `help`?

On the second, I'll do that in `core.py` in the solar system mode PR since I'm making several alterations to that file there, anyway.